AutomatedTesting/Gem/PythonTests/Multiplayer/TestSuite_Main.py

```diff
@@ -20,12 +20,36 @@ class TestAutomation(EditorTestSuite):
     class test_Multiplayer_AutoComponent_NetworkInput(EditorSingleTest):
         from .tests import Multiplayer_AutoComponent_NetworkInput as test_module
 
+        @classmethod
+        def setup(cls, instance, request, workspace, editor, editor_test_results, launcher_platform):
+            level_cache_folder_path = os.path.join(workspace.paths.platform_cache(), "levels", "multiplayer", "autocomponent_networkinput")
+            if os.path.exists(level_cache_folder_path):
```

Comment:
I would maybe just suggest logging a debug/warning message if the folder is not found for whatever reason

Reply:
Updated! Calling _pytest.fail_ if the asset cache wasn't found.